source/core/ut_utils.pkb

```diff
@@ -492,5 +492,37 @@ procedure append_to_clob(a_src_clob in out nocopy clob, a_clob_table t_clob_tab,
     return to_char(a_value, gc_number_format, 'NLS_NUMERIC_CHARACTERS=''. ''');
   end;
 
+  function trim_list_elements(a_list IN ut_varchar2_list, a_characters_to_trim in varchar2) return ut_varchar2_list is
+    l_trimmed_list ut_varchar2_list := ut_varchar2_list();
+    l_index integer;
+  begin
+    l_index := a_list.first;
+
+    while (l_index is not null) loop
+      l_trimmed_list.extend;
+      l_trimmed_list(l_trimmed_list.count) := regexp_replace(a_list(l_index), '(^['||a_characters_to_trim||']*)|(['||a_characters_to_trim||']*$)');
```

Comment:
I'd build the filter outside of loop into a `l_fiter` variable. Without that, you're concatenating inside a loop every time the loop gets executed.